Firestore/core/test/unit/FSTGoogleTestTests.mm

```diff
@@ -139,7 +139,7 @@ @interface GoogleTests : XCTestCase
  * These members are then joined with a ":" as googletest requires.
  *
  * @see
- * https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md
+ * https://github.com/google/googletest/blob/main/docs/advanced.md
```

Comment:
Note: The previous link is no longer valid. I _think_ this is the right replacement.